www/front_src/src/Resources/models.ts

```diff
@@ -71,15 +64,18 @@ export interface Acknowledgement {
 }
 
 export interface ResourceEndpoints {
-  details: string;
-  performanceGraph?: string;
-  timeline: string;
+  details: string | null;
```

Comment:
Why use null instead of undefined here?

Reply:
because this parameter is **always** returned by the API and is null or string